src/provider/cloud-element-templates/Validator.js

```diff
@@ -22,9 +34,34 @@ export class Validator extends BaseValidator {
     let err;
 
     const id = template.id,
-          version = template.version || '_';
+          version = template.version || '_',
+          schema = template.$schema,
+          schemaVersion = schema && getSchemaVersion(schema);
+
+    // (1) compatibility
```

Comment:
Would it not make more sense to first do validations (2) (ie. ensure schema is defined) before checking that the version matches?

Reply:
👍

Reply:
Makes much more sense though 😄